src/components/Sheet/tests/Sheet.test.tsx

```diff
@@ -41,9 +41,12 @@ describe('<Sheet />', () => {
       {mediaQuery: {isNavigationCollapsed: true}},
     );
 
-    expect(sheet.find(CSSTransition).props().classNames).toStrictEqual(
-      BOTTOM_CLASS_NAMES,
-    );
+    expect(sheet.find(CSSTransition).props().classNames).toStrictEqual({
+      enter: 'Bottom enterBottom',
```

Comment:
This is a bit meh, but I think it's a bit better than exposing the BOTTOM_CLASS_NAMES RIGHT_CLASS_NAMES or extracting them out into a file that used by both Sheet and Sheet.test.tsx